pkg/minishift/network/settings_windows.go

```diff
@@ -131,7 +131,8 @@ func ConfigureNetworking(machineName string, networkSettings NetworkSettings) {
 
 	}
 
-	printNetworkSettings(networkSettings)
+	msg := printNetworkSettings(networkSettings)
+	fmt.Println(msg)
```

Comment:
@praveenkumar I guess this is not required

Reply:
@LalatenduMohanty yes this is required.